--- a/FINAL IPYNBS/LSTM.ipynb
+++ b/FINAL IPYNBS/LSTM.ipynb
@@ -39,22 +39,9 @@
     "outputId": "6b7a6e30-55a4-44dd-9991-7a2c1dd56c6f"
    },
    "id": "e0bUJD8BhY42",
-   "execution_count": 18,
-   "outputs": [
-    {
-     "output_type": "stream",
-     "name": "stdout",
-     "text": [
-      "Drive already mounted at /content/drive; to attempt to forcibly remount, call drive.mount(\"/content/drive\", force_remount=True).\n"
-     ]
-    }
-   ]
+   "execution_count": null,
+   "outputs": []
   },
   {
    "cell_type": "code",
-   "execution_count": 19,
+   "execution_count": null,
